Fixed a few typos and changed the sentence structure. (#2842)
umbraco/UmbracoDocs: Umbraco-Heartcore/Getting-Started-Cloud/Creating-a-Heartcore-project/index.md

:::tip
**Media Picker**

This editor lets you upload or select an existing media item from the Media section and add it to your content.
:::

With these properties added our Document Type now looks something like this:

![More properties added to Document Type](images/DocType-More-Properties.png)

> Remember to save the Document Type by clicking the green **Save** button in the bottom-right.

Reviewer: We prefer using "selecting" over "clicking", as "selecting" covers both "clicking" and "tabbing" your way to the button.

```suggestion
Remember to save the Document Type by selecting the green **Save** button in the bottom-right corner.
```

4. Give the content item a name: *Welcome to Umbraco Heartcore*.
5. Add some text to both the *Heading* and the *Intro text* properties.
6. To choose an image for the *PromoImage* property, select the plus-sign.
7. In the dialog, you can do either of the following the choose an image:
    * Select the **Upload** button to choose an image from your file explorer or
    * Drag an image directly into the dialog.

> 8. Once you've chosen an image, select it at the dialog and click the green **Select** button in the bottom of the dialog.

Reviewer: ```suggestion
8. Once you've chosen an image, select it from the dialog and select the green **Select** button at the bottom of the dialog.
```

:::tip
**Media items and images**

When you add media and images to your content using the default Media Picker Data Type, the items will automatically be added to the media library you can find in the **Media** section of the backoffice.

> Select the **Media**  section in the top navigation bar, and you can browse all the media items that have been uploaded to the project.

Reviewer: ```suggestion
Select the **Media** section in the top navigation bar, and you can browse all the media items that have been uploaded to the project.
```

![Blog area - Grid view](images/Blog-grid.png)

> You will now get an overview of the blog posts you've created. So far we've only created one, and creating more is done by clicking on the **Create Blog Post** button in the top-left corner.

Reviewer: ```suggestion
You will now get an overview of the blog posts you've created. So far we've only created one, and creating more is done by selecting the **Create Blog Post** button in the top-left corner.
```
